app/javascript/components/TiptapExtensions/FileEmbedGroup.tsx

```diff
@@ -37,6 +38,15 @@ type FileEmbedGroupStorage = { lastCreatedUid: string | null };
 
 export const titleWithFallback = (title: unknown) => (title ? String(title).trim() : "") || "Untitled";
 
+export const getFilesInGroup = (node: ProseMirrorNode, files: FileEntry[]) => {
```

Comment:
Might make sense to move the memo here as well to match what we do on the download page?